applications/ipc_radio/README.rst

```diff
@@ -76,14 +76,17 @@ The Bluetooth Low Energy and IEEE 802.15.4 functionalities can operate simultane
 Sysbuild Kconfig options
 ========================
 
-To enable the firmware, use the sysbuild configuration of :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO`.
+To enable the firmware, use the sysbuild configuration of :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO`.
 
 You can set the supported radio configurations using the following sysbuild Kconfig options:
 
 * :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC`
 * :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC`
 * :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO_IEEE802154`
 
+.. note::
+   For |NCS| samples and applications the :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO` is used instead of :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO` in order to enable to firmware.
```

Comment:
```suggestion
   For |NCS| samples and applications, use the :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO` sysbuild configuration instead of :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO` to enable to firmware.
```